tests/stub-card-project/searcher.js

```diff
@@ -5,7 +5,7 @@ module.exports = class StubCardSearcher {
   }
 
   constructor(params) {
-    this.cards = params.cardSearchResults;
+    this.cards = params.cardSearchResults || [];
     this.cardIds = this.cards.map(i => i.data.id);
   }
 
```

Comment:
Why is this searcher needed? Is it a placeholder under some other API exists for getting a card?

Reply:
it's used to test that we can implicitly load a card returned by a custom searcher.

Reply:
(we have a similar test for custom indexers too)